.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+<!-- Replace {TICKET} with the ISSUE-### value of the issue number -->
+<!-- OR replace the link with NO TICKET if there was no ticket for the work -->
+[{TICKET}](https://github.com/Tram-One/tram-one-express/issues/{TICKET})
+
+## Problem/Feature
+<!-- Describe the problem this ticket fixes -->
+
+
+## Fix/Implementation
+<!-- Describe your fix -->
+
+
+## Preflight Checklist
+* [ ] Tests added?
+* [ ] PR build is passing?
```

Comment:
This PR template is a little overkill, we don't need a checklist (that is indicated in the PR by the pipeline), and the issue just asks for a `Summary`